src/Entities/ProjectEntity.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace ProjectManager\Entities;
+
+use ProjectManager\Services\DateService;
+
+class ProjectEntity implements \JsonSerializable {
+
+    private int $id;
+    private string $name;
+    private int $client_id;
+    private ?string $deadline;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getClientId(): int
+    {
+        return $this->client_id;
+    }
```

Comment:
do you actually need these getters? doesnt look like your using them anywhere?

Reply:
We are following the diagram and future proofing,